starfish/experiment/experiment.py

```diff
@@ -151,21 +153,38 @@ def __repr__(self):
         return object_repr + fov_repr
 
     @classmethod
-    def from_json(cls, json_url: str) -> "Experiment":
+    def from_json(cls, json_url: str, strict: bool=None) -> "Experiment":
         """
         Construct an `Experiment` from an experiment.json file format specifier
 
         Parameters
         ----------
         json_url : str
             file path or web link to an experiment.json file
+        strict : bool
+            if true, then all JSON loaded by this method will be
+            passed to the appropriate validator
 
         Returns
         -------
         Experiment :
             Experiment object serving the requested experiment data
 
+        Environment variables
+        ---------------------
+        STARFISH_STRICT_LOADING :
+             If set, then all JSON loaded by this method will be
```

Comment:
I think the documentation here is somewhat unclear.  One could read it as:

`STARFISH_STRICT_LOADING` set, but strict=False ==> no validation.

Reply:
Pushed  a clarification.